<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>label_select</class>
 <widget class="QDialog" name="label_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QPushButton" name="pushButton_save">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>550</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>分类标签</string>
      </property>
     </widget>
     <widget class="QListWidget" name="labelType_listWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>256</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="selected_listWidget">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>256</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_wave">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>波形</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_state">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>状态</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>标签列表</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
